apph-back/src/main/java/com/viseo/apph/service/PhotoService.java

```diff
@@ -99,4 +99,16 @@ public PhotoResponse download(Long userId, PhotoRequest photoRequest) throws Fil
         byte[] photoByte = s3Dao.download(photo);
         return new PhotoResponse().setData(photoByte).setTitle(photo.getTitle()).setFormat(photo.getFormat());
     }
+    
+    @Transactional
+    public void deletePhotos(String userLogin, long[] ids) {
+        User user = userDao.getUserByLogin(userLogin);
+        for (long id : ids) {
+            Photo photo = photoDao.getPhoto(id);
+            if (photo != null && photo.getUser().getId() == user.getId()) {
+                s3Dao.delete(s3Dao.hashString(String.valueOf(photo.getId())) + photo.getFormat());
```

Comment:
Il faut rester cohérent avec le reste du S3dao donc lui passer un domain Photo et le laisser hasher le string. Je considère que le hashage  de l'url fait partie de la logique du stockage et non du traitement de nos données c'est pour ça que je ne l'ai pas mis dans le service;
J'ai fait d'autres commentaire avec les refacto a faire pour appliquer ce changement.
```suggestion
                s3Dao.delete(photo);
```